pyscriptjs/tests/unit/base.test.ts

```diff
@@ -0,0 +1,93 @@
+import 'jest';
+
+import { BaseEvalElement } from '../../src/components/base';
+import { pyodideLoaded } from '../../src/stores';
+
+customElements.define('py-base', BaseEvalElement);
+
+describe('BaseEvalElement', () => {
+    let instance: BaseEvalElement;
+
+    beforeEach(() => {
+        instance = new BaseEvalElement();
+    });
+
+    it('BaseEvalElement instantiates correctly', async () => {
+        expect(instance).toBeInstanceOf(BaseEvalElement);
+    });
+
+    // TODO: This test is a bit silly, but the idea is to keep it here until we fix this
+    // issue
+    it("should fail with: Cannot use 'in' operator to search for 'runPythonAsync' in undefined", async () => {
+        let thrownError;
+        let expectedTypeError = new TypeError("Cannot use 'in' operator to search for 'runPythonAsync' in undefined");
+        try {
+            instance.runAfterRuntimeInitialized(async () => {
+                return;
+            });
+        } catch (error) {
+            thrownError = error;
+        }
+        expect(thrownError).toEqual(expectedTypeError);
+    });
```

Comment:
That's a great question(s)!

We could mock `runAfterRuntimeInitialized` similar to what I've done in [pybutton.test.ts](https://github.com/pyscript/pyscript/pull/711/files#diff-c0acae7594b3c4c96e6662f21c5aab8015304903fdd2ef3e95d8721c69487399) so this exception won't impact the tests.

This test was added mostly because I was poking at the exception and trying to figure out what's happening and thought it could be good to include in the PR so that when we fix the issue, his test will fail and we can remove it. Maybe that's a bad idea? :smile:

Reply:
I think it might be the best approach for now. But we can get back to it once Antonio is back on vacation and I hear what he has to say =)